src/test/java/com/microsoft/sqlserver/jdbc/preparedStatement/BatchExecutionWithBulkCopyTest.java

```diff
@@ -228,34 +272,28 @@ public void testMixColumns() throws Exception {
             f1.setAccessible(true);
```

Comment:
can we use the new isSqlAzureDW method now?

Reply:
no, the isSqlAzureDW is what actually checks the server type. We want this test to think that it's using Azure DW through reflection (while it's actually not using a DW)